.changeset/neat-teachers-compare.md

```diff
@@ -0,0 +1,9 @@
+---
+'@leafygreen-ui/combobox': minor
```

Comment:
Why is this a minor version instead of a patch?

Reply:
I was interpreting `Updates focus behavior...` as a feature. Can be convinced otherwise though